Excel-custom-functions/Batching/README.md

```diff
@@ -0,0 +1,80 @@
+# Custom functions in Excel (Preview)
+
+Learn how to use custom functions in Excel (similar to user-defined functions, or UDFs). Custom functions are JavaScript functions that you can add to Excel, and then use them like any native Excel function (for example =Sum). This sample accompanies the [Custom Functions Overview](https://docs.microsoft.com/office/dev/add-ins/excel/custom-functions-overview) topic.
+
+## Table of Contents
+* [Change History](#change-history)
+* [Prerequisites](#prerequisites)
+* [To use the project](#to-use-the-project)
+* [Making changes](#making-changes)
+* [Debugging](#debugging)
+* [IntelliSense for the JSON file in Visual Studio Code](#intellisense-for-the-json-file-in-visual-studio-code)
+* [Questions and comments](#questions-and-comments)
+* [Additional resources](#additional-resources)
+
+## Change History
+
+* Oct 27, 2017: Initial version.
+* April 23, 2018: Revised and expanded.
+* June 1, 2018: Bug fixes.
+
+## Prerequisites
+
+* Install Office 2016 for Windows and join the [Office Insider](https://products.office.com/en-us/office-insider) program. You must have Office build number 10827 or later.
+
+## To use the project
```

Comment:
This heading is out of agreement with the other headings. Change to "Using the Excel custom functions project"